samples/cellular/nrf_cloud_multi_service/sample.yaml

```diff
@@ -94,7 +94,7 @@ tests:
     integration_platforms:
       - nrf5340dk/nrf5340/cpuapp/ns
     platform_allow: nrf5340dk/nrf5340/cpuapp/ns
-    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
```

Comment:
This build is not scan-only

Reply:
(Therefore if I am not mistaken, I think this line should remain unchanged -- it should stay `nrf7002ek`)